client/src/App.vue

```diff
@@ -3,7 +3,8 @@
     div(v-if="hasError")
       ErrorPage
     div(v-else)
-      #background(:style="{ backgroundImage: `url('${imageUrl}')` }")
+      #background(:style="{ backgroundImage: `url('${imageUrl}')` }" v-if="styleOptions")
+      #background(:style="{ backgroundColor: `#000` }" v-else)
```

Comment:
For `v-else` we can remove the style option and just add it directly in the CSS

Reply:
You mean to add a class instead of the inline style?

Reply:
Yep!